arm-containerregistry/2017-03-01/examples/RegistryGet.json

```diff
@@ -0,0 +1,34 @@
+{
+    "parameters": {
+        "api-version": "2017-03-01",
+        "subscriptionId": "subid",
+        "resourceGroupName": "rgname",
+        "registryName": "registry"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/subid/resourceGroups/rgname/providers/Microsoft.ContainerRegistry/registries/registry",
+                "name": "registry",
+                "type": "Microsoft.ContainerRegistry/registries",
+                "location": "westus",
+                "tags": {
+                    "key": "value"
+                },
+                "sku": {
+                    "name": "Basic",
+                    "tier": "Basic"
+                },
+                "properties": {
+                    "loginServer": "registry.azurecr.io",
+                    "creationDate": "2017-03-01T23:14:37.0707808Z",
+                    "provisioningState": "Succeeded",
+                    "adminUserEnabled": true,
```

Comment:
/cc @amarzavery - This is our preview API and as described we are aligning this with preview and we cannot change our backed implementation at this point. We will accommodate this change in our next api version.